tox.ini

```diff
@@ -0,0 +1,27 @@
+
+[tox]
+envlist =
+       {py27,py35}-django{19,110},
+       {py27,py35,py36}-django111,
+       {pypy}-django111,
+skip_missing_interpreters = True
+
+[travis:env]
+DJANGO =
+    1.8: django18
+    1.9: django19
+    1.10: django110
```

Comment:
same here, I think we can remove everything except 1.11 and then maybe add 2.0 and 2.1